xcodebuild/xcodebuild.go

```diff
@@ -158,19 +158,19 @@ func createXcodebuildOptions(xctestrun string, onlyTesting, skipTesting []string
 	}
 
 	if 0 < len(onlyTesting) {
-		var args []string
+		var args string
```

Comment:
Maybe add a comment documenting this weird stuff so that we don't refactor this code accidentally?